metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -1185,6 +1185,15 @@ def _process_view(
         except NotImplementedError:
             description: Optional[str] = None
             properties: Dict[str, str] = {}
+        except ProgrammingError as pe:
+            # Snowflake needs schema names quoted when fetching table comments.
+            logger.debug(
+                f"Encountered ProgrammingError. Retrying with quoted schema name for schema {schema} and view {view}",
```

Comment:
I simply followed this code for getting table comments - https://github.com/datahub-project/datahub/blob/v0.8.38/metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py#L999-L1007 and modified it for view, because I was getting the same error for view. 
I think, its better off in except  clause since we wouldn't want to apply this for all other sql-common sources.

To give an example -  for schema with name `public`, quotes are not needed, whereas quotes are needed for schema with name `test-schema`
`show tables like 'tblname' in  schema public`
`show tables like 'tblname' in  schema "test-schema"`